### ISSUE TITLE
Possible removal of boilerplate code using Lombok. 

### ISSUE THREAD

**Issue by AarushJ** [Cyber-Labs/ism-app-android#1]
Please Check if Lombok package is compatible with Android. 
If it is compatible, use it for Getters, Setters, Constructors etc. 
Also use Lombok for logging if it is compatible with Android. Espcially, **@Slf4j** annotation.   

**See** 
 [Lombok Project](https://projectlombok.org/ "Lombok-project")

**TODO**
If Lombok is supported, then use as much of it as you can to reduce the boilerplate code in the already pushed code. 
If it is not supported, then notify and I will close this issue.

**Comment by ujjwalagrawal17** [comment]
@AarushJ  yes it supported on Android. @peruljain @amanatgithub please check and try to implement it. Let me know if you are facing any issues in integrating it.
